public/video-ui/src/services/VideosApi.js

```diff
@@ -121,12 +121,15 @@ export default {
 
     function updateArticleField(stage, data, composerUrl, pageId) {
       if (data.value || data.belongsTo === 'settings') {
+        const value = data.isHtml
+          ? data.value.split('"').join('\\"')
```

Comment:
Yes, although mentioning this made me realise that having double quotes in any text fields (e.g. title) would break syncing back to composer so I'm fixing this to do it for all the fields wit text input.